src/components/navi-button/navi-button.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { NaviButtonProps } from './types';
+import clsx from 'clsx';
+import { Label, LabelSize } from '../typography';
+import { BaseButton } from '../../base/base-button/base-button';
+import { IconSize } from '../../elements/icons/types';
+
+export const NaviButton: FC<NaviButtonProps> = ({ onClick, className, Icon, label }) => {
+    const borderButtonClasses = clsx(
+        'flex flex-row items-center justify-center',
+        'h-xxl w-xxl',
+        'hover:cursor-pointer',
+        'group rounded-s',
+        className,
+    );
+
+    const iconClasses = clsx('self-center fill-white');
+    const isIconSettings =
```

Comment:
Maybe add a new Icon called `IconSettingsAnimated` or similar, so we can re-use it and also have less logic in the navi-button

Reply:
same as you did with IconLogoutAnimated